_data/stage3.yml

```
@@ -492,7 +492,7 @@
 - title: Promise.any
   id: proposal-promise-any
   has_specification: true
-  description: Introduce String.prototype.replaceAll as a way to replace all in a string
+  description: Adds a method which returns a promise that is fulfilled by the first given promise to be fulfilled, or rejected if all of the given promises are rejected.
```

Well, I was mentally nitpicking whether this was a method or a function, but then I realized it didn't really matter. 

A bit later, I realized that if it doesn't matter, it might as well get trimmed from the page, too:

```suggestion
  description: Returns a promise that is fulfilled by the first given promise to be fulfilled, or rejected if all of the given promises are rejected.
```

I won't lose any sleep if you don't like this change, though.

---

Haha, I appreciate your conviction for brevity. I think technically it is a static method, though, so I'm going to leave the extra words.